api/candlepin-api-spec.yaml

```diff
@@ -1343,6 +1517,90 @@ components:
             upstreamConsumer:
               $ref: '#/components/schemas/UpstreamConsumerDTO'
 
+    ProductContentDTO:
+      description: Product content mapping exposed to the API
+      required:
+        - id
```

Comment:
this DTO does not have an id field, so the required section should be removed

Reply:
Done.